### ISSUE TITLE
An ADA issue: the keyboard focus is not visible for the home page's carousel buttons

### ISSUE THREAD

**Issue by dmitry-fedyuk** [kitchencollection-com/core#24]
[upwork.com/ab/f/contracts/20874278](https://www.upwork.com/ab/f/contracts/20874278)

![01 - copy](https://user-images.githubusercontent.com/83204/47472740-959ea780-d831-11e8-8f2d-08e3186301c9.png)

**Comment by dmitry-fedyuk** [comment]
Fixed in [0.1.4](https://github.com/kitchencollection-com/core/releases/tag/0.1.4).
